bin/mail/2017-teacher-application-responses/test_teacher_application_decision_processor.rb

```diff
@@ -89,7 +90,7 @@ def test_process_decisions_row_accept_teachercon
       name: 'Tracy Teacher',
       email: 'tracy.teacher@example.net',
       preferred_first_name_s: 'Tracy',
-      course_name_s: 'CS Principles',
+      course_name_s: 'CS Discoveries',
```

Comment:
Maybe we should avoid hardcoding this string any more than necessary in Ruby code, and instead just reference the constant